src/js/http_outgoing.js

```diff
@@ -17,14 +17,18 @@
 var util = require('util');
 var stream = require('stream');
 
+var crlf_buf = Buffer.from('\r\n');
+
 
 function OutgoingMessage() {
   stream.Stream.call(this);
 
   this.writable = true;
 
   this._hasBody = true;
-  this._bodyLength = 0;
+  this._contentLength = null;
```

Comment:
Why removing the `_bodyLength`?

Reply:
It's renamed for align with naming of the header `Content-Length`.

Reply:
Good explain! 👍